src/pages/PersonalInfo/PersonalInfo.tsx

```diff
@@ -1,8 +1,33 @@
+import { useState, useEffect } from 'react';
+import styles from './PersonalInfo.module.css'
+import inter from './'
+
 const PersonalInfo = () => {
   // Personal info tab
   console.log("PersonalInfo");
+  const [tabIndex, setTabIndex] = useState(0);
+  const tabs = ["One", "Two", "Three", "Four", "Five"];
+
+  return (
+    <div className={styles.tabs}>
+      <link rel="stylesheet"
```

Comment:
Move this into the `<header>` sections of the `index.html` file in the root folder so the font applies to the whole project.